snapcraft/storeapi/errors.py

```diff
@@ -396,6 +410,16 @@ def __fmt_error_401_or_403(self, response):
     def __fmt_error_404(self, response):
         return self.__FMT_NOT_REGISTERED
 
+    def __fmt_error_500(self, response):
+        response_json = self.__to_json(response)
+        message = ('The store encountered an internal error. Please report a '
+                   'bug if this continues.')
```

Comment:
where?

Reply:
also reporting a bug without much information or the fact that it is repeatable would be a really hard to fix thing. This should probably be something done with sentry.
I would state that if the error persists, to log a bug or reach out for support, but not on the first try (we could also tell them to look at the status page)

Reply:
Alright, done.